compute/admin_guide/continuous_integration/jenkins_pipeline_project.adoc

```diff
@@ -73,10 +70,15 @@ Note that the previous scan step already published the results in Console, and t
 
 .. In the *Sample Step* drop-down, select *prismaCloudPublish - Publish Prisma Cloud analysis results*.
 
-.. In *Scan Result Files*, accept the default.
+.. In *Scan Result Files*, review the json filename.
++
+If you have configured scanning for multiple images and configured unique filenames for each scan in the previous step, you must add a wildcard to the json filename for scan results. For example `prisma-cloud-scan-results*.json`. This ensures that the results of the scan are not overwritten and you can view it in the Jenkins plugin.
```

Comment:
@Pubs-MV, maybe we can change it from "This ensures that the results of the scan are not overwritten and you can view it in the Jenkins plugin." to something like:
"This ensures that the publish command will read all the result files with the same name pattern, and publish them"

Reply:
@MorShabiPAN done. Please approve for publish.